### PR TITLE
Update README.md to support Windows

### PR DESCRIPTION
**Pull request by lucasterra**
You need to use use [\/\\] instead of \/ to match folder separators on Windows.

[Review on Reviewable](https://reviewable.io/reviews/shakacode/bootstrap-loader/44)

### CODE REVIEW

**Comment by justin808** [comment]
@alexfedoseev I think we should suggest either Unix or Windows versions, or a universal one. The universal one might be a bit harder for folks to read.

File: README.md [hunk 1]
```diff
@@ -289,10 +289,10 @@ module: {
     // Use one of these to serve jQuery for Bootstrap scripts:
 
     // Bootstrap 3
-    { test: /bootstrap-sass\/assets\/javascripts\//, loader: 'imports?jQuery=jquery' },
+    { test:/bootstrap-sass[\/\\]assets[\/\\]javascripts[\/\\]/, loader: 'imports?jQuery=jquery' },
```
Comment (justin808) [inline README.md]: We need separate sections to explain windows vs. non-windows. The backslash slash escaping is just too much for mere mortals.

**Comment by justin808** [comment]
@lucasterra Any chance you can update your PR?

**Comment by lucasterra** [comment]
The regex I suggested is universal. It works on both: Unix and Windows.
I believe it's less confusing to have a universal than separate versions for each system.

But sure, I can update it if you want.

**Comment by justin808** [comment]
@lucasterra Given your suggestion, I'd say, let's offer the simpler, Unix one, and a more sophisticated universal one.

**Comment by justin808** [comment]
@lucasterra Any word on this one?

**Comment by lucasterra** [comment]
I think it would be simpler to offer just an universal one. It's just a
Regex... It's not like it would super complicate things.

Offering one that works on all systems sounds simpler than saying: "You're
on Windows? Use this. You on Unix? Use that"

Em seg, 14 de mar de 2016 às 23:38, Justin Gordon <notifications@github.com>
escreveu:

**Pull request merged**